src/main/java/ch/uzh/ifi/ase/csccrecommender/recommender/DocumentComparison.java

```diff
@@ -8,53 +8,107 @@
 
 public class DocumentComparison {
 
+  private static final int LINE_CONTEXT_SWITCH_THRESHOLD = 25;  // 25 is picked randomly
   private final Document document;
-  private final long overallContextHammingDistance;
-  private final long lineContextHammingDistance;
-  private Integer overallContextLcsDistance = null;
-  private Integer lineContextLevenshteinDistance = null;
+  private final long overallContextHammingDistance; //Could be converted into local variable
```

Comment:
Just remove it, if you think it's not needed anymore :)

Nobody will look at these comments anymore in the future.

Reply:
Sure, you are the author of this class so I'm waiting for your comment :)